lokalise.sh accidentally exits; #trivial

jotshin opened the pull request:
Jira ticket: https://conichi.atlassian.net/browse/CNI-8698

## Description
When adding strings to lokalise.co, I usually use this script to add the strings. However, sometimes it might fail and I have to do `source ~/.bashrc` to make it work.

This bug happens due to `set -e` will make the script exits when it can't find the environment variables set in `.bashrc` file, as the reason is pointed out below:

> export exports the variable assignment to sub-shells, i.e. shells which are started as child processes of the shell containing the export directive. Your command-line environment is the parent of the script's shell, so it does not see the variable assignment.
from https://askubuntu.com/questions/53177/bash-script-to-set-environment-variables-not-working



Reference: https://stackoverflow.com/questions/19622198/what-does-set-e-mean-in-a-bash-script

ShengHuaWu commented on scripts/lokalise.sh:
```diff
@@ -1,7 +1,5 @@
 #!/bin/bash
 
-set -e
```
Could you explain what `set -e` means? 🤔

jotshin replied:
from `set -e` was an attempt to add "automatic error detection" to the shell. Its goal was to cause the shell to abort any time an error occurred, so you don't have to put || exit 1 after each important command.

jotshin commented:
@David-Henner @ShengHuaWu Guys, sorry for my original bad fix. I've already updated the description to make clearer and updated the fix. Please review it. Thanks.
